Cluster-Backup/Kasten-Minikube/Mac-delete.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+export NAMESPACE=kasten-io
+
+
```

Comment:
You will need to add the following 2 commands as well to remove the helm repo and delete all the resources 

helm repo remove kasten

kubectl delete all --all -n $NAMESPACE